api/v1/helpers/verbs/doPut.js

```diff
@@ -70,9 +70,10 @@ function doPut(req, res, next, props) {
             // take nullified fields out of changed fields
             o.changed(key, false);
           } else {
-
-            // value may have changed. set changed to true to
-            // trigger checks in the model
+            /**
+             * value may have changed. set changed to true to
+             * trigger checks in the model
+             */
             o.changed(key, true);
```

Comment:
Is this really necessary here? (I thought we were doing this just for samples and subjects in order to make sure we generated the realtime events.)